mkdocs.yml

```diff
@@ -0,0 +1,6 @@
+site_name: Online Ratings
```

Comment:
Actually stuck with what you suggested.  Didn't notice this was the mkdocs file when I initially replied.  I think this part of the repo should eventually be documentation for all of online-ratings.